backend/src/main/java/com/bakdata/conquery/models/forms/arx/AttributeTypeBuilder.java

```diff
@@ -68,16 +84,127 @@ public AttributeType build() {
 				extended[extended.length - 1] = SUPPRESSING_GRANULARITY;
 				hierarchy[i] = extended;
 			}
+
+			// Add hierarchy handling for empty values
 			String[][] newhierarchy = Arrays.copyOf(hierarchy, hierarchy.length + 1);
 			String[] emptyValueHierarchy = new String[hierarchy.length > 0 ? hierarchy[0].length : 0];
 			Arrays.fill(emptyValueHierarchy, "");
-			emptyValueHierarchy[emptyValueHierarchy.length - 1] = "*";
+			emptyValueHierarchy[emptyValueHierarchy.length - 1] = SUPPRESSING_GRANULARITY;
 			newhierarchy[newhierarchy.length - 1] = emptyValueHierarchy;
 
 			return AttributeType.Hierarchy.create(newhierarchy);
 		}
 	}
 
+	class ConceptHierarchyNodeId implements AttributeTypeBuilder {
```

Comment:
ist das eigentlich absichtlich nicht static?

Reply:
guter punkt, das sollte static sein. Danke dir!

Reply:
habe gerade nochmal nachgeschaut: da es in einem public interface geschachtelt ist sind die Modifikatoren  hinfällig